### PR TITLE
Free moving item

### PR DESCRIPTION
**Pull request by valdisz**
Man-made monsters,

Ported from Magic Deep with two items: catapult and steel defender.

### CODE REVIEW

File: items.cpp [hunk 1]
```diff
@@ -784,6 +804,53 @@ AString *ItemDescription(int item, int full)
...
+			if (regen > 0) {
+				*temp += AString(" This FMI regenerates ") + regen +
+					" hits per round of battle.";
+			}
+			*temp += AString(" This FMI has a tactics score of ") +
+				mp->tactics + ", a stealth score of " + mp->stealth +
+				", and an observation score of " + mp->obs + ".";
+		}
+
+		if (mp->spoiltype != -1) {
```
Comment (artyomtrityak) [inline items.cpp:839]: Why free moving items have spoils?
As a something players produce we might not want it to drop anything right?

**Review by artyomtrityak — COMMENTED** [review]
This looks pretty good! can you please change PR towards  neworigins-v3 branch and it will be good to merge!

File: items.cpp (hunk 1 above)
Comment (valdisz) [reply items.cpp:839]: Will be there spoils or not depends on item definition. In the sample catapult and steel defender `spoiltype` is set to have 0 spoils.

**Comment by valdisz** [comment]
> This looks pretty good! can you please change PR towards neworigins-v3 branch and it will be good to merge!

Ok.

**Review by artyomtrityak — APPROVED** [review]

**Pull request merged**